lean/models/config.py

```diff
@@ -91,7 +92,8 @@ def get_internal_name(self) -> str:
         """
         return {
             DebuggingMethod.PyCharm: "PyCharm",
-            DebuggingMethod.PTVSD: "PTVSD"
+            DebuggingMethod.PTVSD: "PTVSD",
+            DebuggingMethod.QC: "PTVSD" # QC -> PTVSD, If its Python it uses PTVSD, if its C# it is irrelevant
```

Comment:
This is important, because the backend is PTVSD for Python and NetCoreDbg for C#. But they are going to be connected to by the QC Extension